fdbserver/MasterProxyServer.actor.cpp

```diff
@@ -1804,17 +1804,17 @@ ACTOR Future<Void> forwardProxy(ClientDBInfo info, PromiseStream<ReplyPromise<Co
 	loop {
 		choose {
 			when(ReplyPromise<CommitID> req = waitNext(commitReplies.getFuture())) {
-				CommitID rep;
+				CommitID rep{};
```

Comment:
CommitID has constructor, so probably right place to fix is over there.

Reply:
Turns out CommitID's default constructor already initializes, and GetReadVersionReply is the only problematic one here.

FWIW I actually kind of like explicitly initializing memory, because it means that valgrind can give you really nice diagnostics (with `--track-origins=yes`) for this kind of bug. But for this to be a good idea we'd also probably need to run valgrind on every PR

Reply:
Yeah I agree with you on having nice stuff like that.

I just meant from consistency point of view. Cause people tend to look at code around lot of times and follow that pattern, and if I see some message `ABCReply rep` using constructor around I'm more likely to use `GetReadVersionReply rep;` instead of `GetReadVersionReply rep{}` and make a mistake. But if we use one pattern for all messages its harder to make such mistakes, and probably report produced by valgrind be more useful.

Reply:
I now initialize GetReadVersionReply in its default constructor. PTAL